Week1/prep-exercises/1-catwalk-promises/index.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const STEP_SIZE_PX = 10;
+const STEP_INTERVAL_MS = 50;
+const DANCE_TIME_MS = 5000;
+const DANCING_CAT_URL =
+  'https://media1.tenor.com/images/2de63e950fb254920054f9bd081e8157/tenor.gif';
+
+function walk(img, startPos, stopPos) {
+  return new Promise((resolve) => {
+    // Resolve this promise when the cat (`img`) has walked from `startPos` to
+    // `stopPos`.
+    // Make use good use of the `STEP_INTERVAL_PX` and `STEP_INTERVAL_MS`
```

Comment:
Forgot to correct this typo.

```suggestion
    // Make good use of the `STEP_INTERVAL_PX` and `STEP_INTERVAL_MS`
```